model/src/main/resources/db-update/20060-from-2.5.0-to-2.6.0.sql

```diff
@@ -67,4 +67,64 @@ BEGIN;
     ALTER TABLE buildrecord ADD COLUMN scmbuildconfigrevision varchar(255);
     ALTER TABLE buildrecord ADD COLUMN scmbuildconfigrevisioninternal boolean;
 
+--------------------------------------------------------------------------------
+-- DeliverableAnalyzerReport
+--------------------------------------------------------------------------------
+CREATE TABLE delan_report
+    (
+        operation_id        BIGINT                  NOT NULL,
+        label               VARCHAR(255),
+        PRIMARY KEY (operation_id)
+    );
+    ALTER TABLE delan_report
+        ADD CONSTRAINT fk_delanreport_operation
+            FOREIGN KEY (operation_id)
+            REFERENCES operation(id);
+
+--------------------------------------------------------------------------------
+-- DeliverableArtifact
+--------------------------------------------------------------------------------
+CREATE TABLE deliverable_artifact
+    (
+        report_id           BIGINT                  NOT NULL,
+        artifact_id         INTEGER                 NOT NULL,
+        built_from_source   BOOLEAN,
```

Comment:
```suggestion
        built_from_source   BOOLEAN                 NOT NULL,
```

Reply:
Oh yes, we can afford to have NOT NULL even here, since `DeliverableArtifact#builtFromSource` is of type `boolean` (and not `Boolean`), see 5213c27